scripts/core/superdesk-api.d.ts

```diff
@@ -1816,6 +1816,15 @@ declare module 'superdesk-api' {
         config: IConfig;
     }
 
+    export interface ITemplateEditorComponentProps<IValue, IConfig> {
+        item: IArticle;
+        value: IValue;
+        setValue: (value: IValue) => void;
+        readOnly: boolean;
+        config: IConfig;
+        radioValue?: string,
```

Comment:
It's a generic interface meant to suit any authoring field, there should be no fields in the object that are specific to one authoring field.

It looks to me like you should be able to compute radio value from field value.